### ISSUE TITLE
Failed to build module 'Razorpay' from its module interface; the compiler that produced it, 'Apple Swift version 5.5.1 (swiftlang-1300.0.31.4 clang-1300.0.29.6)', may have used features that aren't supported by this compiler, 'Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)'

### ISSUE THREAD

**Issue by Akash-Neeli** [razorpay/razorpay-flutter#213]
[✓] Flutter (Channel stable, 2.5.0, on Mac OS X 10.15.7 19H1417 darwin-x64, locale en-SG)
    • Flutter version 2.5.0 at /Users/gourav/Developer/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4cc385b4b8 (3 months ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/gourav/Library/Android/sdk
    • Platform android-31, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] VS Code (version 1.63.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.29.0

[✓] Connected device (2 available)
    • Adaptabiz's iPad (mobile) • 00008020-001E309E01F0402E • ios            • iOS 14.7.1 18G82
    • Chrome (web)              • chrome                    • web-javascript • Google Chrome 96.0.4664.93
    ! Error: Adaptabiz's iPad is busy: Copying cache files from device. Xcode will continue when Adaptabiz's iPad is finished. (code -10)

• No issues found!

**Comment by sanjaymajoka** [comment]
Hi I faced same issue. To solve this I downloaded xcode 13.1 (comes with swift 5.5.1) and updated my Mac OS. But still no success. 

But after removing my old xcode (had swift 5.4) my iOS app build Successfully.

**Comment by Akash-Neeli** [comment]
> Hi I faced same issue. To solve this I downloaded xcode 13.1 (comes with swift 5.5.1) and updated my Mac OS. But still no success.
> 
> But after removing my old xcode (had swift 5.4) my iOS app build Successfully.

Could you please specify which xcode Version, Macos Version and razorpay version you are using ?

**Comment by shrutika-tatkare-apptware** [comment]
Do you found any solution ?

**Comment by Akash-Neeli** [comment]
> Do you found any solution ?

Yes, I've run the project in Latest Xcode 13 and it is working Fine

**Comment by hardikvij195** [comment]
@Akash-Neeli  not working for me

**Comment by Akash-Neeli** [comment]
> @Akash-Neeli not working for me

Could you post your error log?

**Comment by sanjaymajoka** [comment]
Check this closed issue.
https://github.com/razorpay/razorpay-flutter/issues/211

**Comment by mhdshuhaib1704** [comment]
> > Do you found any solution ?
> 
> Yes, I've run the project in Latest Xcode 13 and it is working Fine

Which version is od Xcode used before ?
